Package/PluginInstaller.cs

```diff
@@ -130,23 +131,59 @@ IEnumerable<string> possiblePaths(string file)
                 // If OPENTAP_COLOR is set, the escape symbols for colors in the child process will break the parsing of the forwarded logs.
                 // Ensure color is never set in the child process. Colors will still be set in the parent process.
                 pi.Environment["OPENTAP_COLOR"] = "never";
-
+                
                 try
                 {
                     Process p;
                     if (step.UseShellExecute)
                     {
                         pi.UseShellExecute = true;
 
-                        p = Process.Start(pi);
+                        try
+                        {
+                            p = Process.Start(pi);
+                        }
+                        catch (Exception e)
+                        {
+                            // Win32Exception is when EXE was not found.
+                            // InvalidOperationException is when no file name was specified.
+                            // ArgumentException is when the file path is not valid.
```

Comment:
I think we should explicitly handle these two cases before we try to start the program, and just catch the `Win32Exception` for when the executable was not in PATH

Reply:
In fact, it might be better if we handle those two cases at `Package Create` time, rather than at `package install` time.